messaging/src/main/java/io/lantern/messaging/CryptoWorker.kt

```diff
@@ -244,7 +246,9 @@ internal class CryptoWorker(
                             it.attachmentsPendingEncryption.forEach { (id, attachment) ->
                                 val storedAttachment = attachment.toBuilder()
                                 try {
-                                    messaging.encryptAttachment(storedAttachment)
+                                    messaging.encryptAttachment(
```

Comment:
CryptoWorker now does the encryption for self-messages just like for others, except that the status gets updated to DONE instead of PENDING_UPLOAD.